orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/TerminateInstancesTask.groovy

```diff
@@ -45,8 +47,14 @@ class TerminateInstancesTask extends AbstractCloudProviderAwareTask implements T
         "terminate.account.name": account,
         "terminate.region"      : stage.context.region,
         "kato.last.task.id"     : taskId,
-        "kato.task.id"          : taskId, // TODO retire this.
-        "terminate.instance.ids": stage.context.instanceIds,
+        "terminate.instances"   : stage.context.instances,
     ])
   }
+
+  static Map convert(Stage stage) {
+    def operation = [:] << stage.context
+    def instances = operation.remove("instances")
+    operation.instanceIds = instances.collect { it.id }
```

Comment:
So you're going to make multiple calls if the instances fall within different regions/zones? 

Personally, I'd prefer to keep the Clouddriver operate on a single instance, and make Orca coordinate multiple calls, one for each instance.